.circleci/config.yml

```diff
@@ -21,7 +21,11 @@ jobs:
                 name: Build and unzip war
                 command: |
                     mvn -DskipTests clean install && \
-                    unzip portal/target/cbioportal*.war -d portal/target/war-exploded
+                    mkdir portal/target/war-exploded && \
```

Comment:
unzip isn't in the image, nor is apt/yum/etc. This seems to be the equivalent command